src/drivers/camera_trigger/camera_trigger_params.c

```diff
@@ -112,7 +112,8 @@ PARAM_DEFINE_INT32(TRIG_MODE, 0);
 /**
  * Camera trigger pin
  *
- * Selects which pin is used, ranges from 1 to 6 (AUX1-AUX6 on px4_fmu-v2 and the rail
+ * Selects which FMU pin is used, ranges from 1 to 6 (AUX1-AUX6 on Pixhawk controllers
+ * with an I/O board, MAIN1-MAIN6 on controllers without an I/O board, and the rail
```

Comment:
@bkueng Can you explain what this means by "and the rail pins on px4_fmu-v4"? ie are these AUX/MAIN, or something different?

Reply:
It means MAIN (FMU) pins on Pixracer et. al.

Reply:
Thanks @mhkabir . I've now removed that as it is covered by the other "more general" text.